src/plugins/dashboard/public/plugin.tsx

```diff
@@ -308,6 +316,47 @@ export class DashboardPlugin
       stopUrlTracker();
     };
 
+    const registerDashboardProvider: RegisterDashboardProviderFn = (
+      provider: DashboardProvider
+    ) => {
+      const found = this.dashboardProviders[provider.savedObjectsType];
+      if (found) {
```

Comment:
Is there a more-terse way to do these validations?  Typescript interfaces are not catching empty-string on these properties.